operating/manage-vulnerabilities/common-vuln-management-tasks.adoc

```diff
@@ -1,45 +1,82 @@
 
-include::modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc[leveloffset=+1]
+The *Vulnerability Management* -> *Workload CVEs* page provides information about vulnerabilities in applications running on clusters in your system. You can view vulnerability information across images and deployments. The *Workload CVEs* page provides advanced filtering capabilities, including the ability to view images and deployments with vulnerabilities and filter by image, deployment, namespace, cluster, CVE, component, and component source.
 
-include::modules/identify-dockerfile-line-component-cve.adoc[leveloffset=+1]
+//Viewing CVEs
+include::modules/vulnerability-management20-view-cve.adoc[leveloffset=+1]
+
+//Viewing vulns in nodes
+include::modules/identify-vulnerabilities-in-nodes-vm20.adoc[leveloffset=+1]
+include::modules/disable-identify-vulnerabilities-in-nodes.adoc[leveloffset=+2]
+.Additional resources
+* xref:../../operating/examine-images-for-vulnerabilities.adoc#supported-operating-systems_examine-images-for-vulnerabilities[Supported operating systems]
+
+//viewing vulns in platforms
+
+include::modules/vulnerability-management20-view-platform-cve.adoc[leveloffset=+1]
+
+[id="excluding-CVEs"]
+== Excluding CVEs
 
-include::modules/view-details-only-for-fixable-cves.adoc[leveloffset=+1]
+You can exclude or ignore CVEs in {product-title-short} by snoozing node and platform CVEs and deferring or marking node, platform, and image CVEs as false positives. You might want to exclude CVEs if you know that the CVE is a false positive or you have already taken steps to mitigate the CVE. Snoozed CVEs do not appear in vulnerability reports or trigger policy violations.
 
-include::modules/identify-operating-system-of-the-base-image.adoc[leveloffset=+1]
+You can snooze a CVE to globally ignore it for a specified period of time. Snoozing a CVE does not require approval.
 
-include::modules/identify-top-risky-objects.adoc[leveloffset=+1]
+[NOTE]
+====
+Snoozing node and platform CVEs requires enabling the ROX_VULN_MGMT_LEGACY_SNOOZE environment variable.
+====
 
-include::modules/top-risky-images-components.adoc[leveloffset=+1]
+Deferring or marking a CVE as a false positive is done through the exception management workflow. This workflow provides the ability to view pending, approved, and denied deferral and false positive requests. You can scope the CVE exception to a single image, all tags for a single image, or globally across all images.
 
-include::modules/view-dockerfile-for-image.adoc[leveloffset=+1]
+When approving or denying a request, you must add a comment. A CVE remains in the observed status until the exception request is approved. A pending request for deferral that is denied by another user is still visible in reports, policy violations, and other places in the system, but is indicated by a *Pending exception* label next to the CVE when visiting *Vulnerability Management* -> *Workload CVEs*.
```

Comment:
```suggestion
If you approve or reject a request, you must add a comment. A CVE remains in the observed status until the exception request is approved. A pending deferral request that is rejected by another user is still visible in reports, policy violations and elsewhere in the system, but is indicated by a *Pending exception* label next to the CVE when you click *Vulnerability Management* -> *Workload CVEs*.
```

Reply:
The language here is specific because requests are denied (not rejected). Also after "policy violations" the Oxford comma is necessary per IBM Style. I don't think "when you click" is needed to replace "visiting" since the idea is that anyone going to this screen will see the labels.